joplin/pages/home_page/models.py

```diff
@@ -1,15 +1,54 @@
+import re
 from django.db import models
-
+from django.conf import settings
 from wagtail.core.models import Page
+from wagtail.admin.edit_handlers import FieldPanel
+
 
 class HomePage(Page):
     parent_page_types = []
-    # subpage_types = ['base.ServicePage', 'base.ProcessPage', 'base.InformationPage', 'base.DepartmentPage']
-    subpage_types = [
-        'service_page.ServicePage',
-        'information_page.InformationPage',
-        'department_page.DepartmentPage',
-        'topic_page.TopicPage',
-        'location_page.LocationPage',
-        'event_page.EventPage'
+    subpage_types = []
```

Comment:
For the sake of tests, we no longer want to exclude JanisBasePage as a child. At that point, I'm not sure if we need to be explicit about what subpage_types are allowed.